lib/get-header.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function getHeader(data) {
+  var header = {};
+
+  console.log('Getting header...');
+
+  header.head = data.toString('utf-8', 0, 2);
+  header.pixelArrayStart = data.readUInt32LE(10);
+  header.paletteStart = 54;
+  header.paletteSize = data.readUInt32LE(46);
```

Comment:
bunch of magic numbers here. maybe not all of them need to be in constants but at least some of them should be.